### ISSUE TITLE
ServiceBinding userInfo changes after creation

### ISSUE THREAD

**Issue by wozniakjan** [SAP/sap-btp-service-operator#56]
When I create `ServiceBinding`, the `userInfo` section is correctly represented as
```yaml
  userInfo:
    groups:
    - mps-logging-admin
    - mps-monitoring-viewer
    - skr-logging-admin
    - mps-monitoring-admin
    - runtimeAdmin
    - runtimeOperator
    - skr-monitoring-admin
    - skr-logging-viewer
    - mps-logging-viewer
    - skr-monitoring-viewer
    - system:authenticated
    username: jan.wozniak@sap.com
```

But after few reconciles, it changes to `btp-operator` service account before it pronounces the bindings as ready.
```yaml
  userInfo:
    groups:
    - system:serviceaccounts
    - system:serviceaccounts:sap-btp-operator
    - system:authenticated
    uid: 3d2203e0-5b76-4177-9b7f-ed6b0b878264
    username: system:serviceaccount:sap-btp-operator:default
```

[sb.watch.yaml.log](https://github.com/SAP/sap-btp-service-operator/files/6661980/sb.watch.yaml.log)
[btp-operator.log](https://github.com/SAP/sap-btp-service-operator/files/6661984/btp-operator.log)

**Comment by pavelmaliy** [comment]
This [PR](https://github.com/SAP/sap-btp-service-operator/pull/57) will fix it.
You right user info should not be changed by reconciliations that caused by operator

**Comment by pavelmaliy** [comment]
you can take the latest release there this issue is resolved (v0.1.9)
